heartbeat/monitors/util.go

```diff
@@ -274,8 +274,8 @@ func makeByHostAnyIPJob(
 		resolveStart := time.Now()
 		ip, err := net.ResolveIPAddr(network, host)
 		if err != nil {
-			resolveErr(event, host, err)
-			return nil, nil
+			resolveErr(event, host)
```

Comment:
I think that during the refactor my thought was that we'd add the error fields in `resolveErr` directly, which was obviously incorrect.